frontend/src/helpers/nameHelper.ts

```diff
@@ -41,3 +42,23 @@ export function renameServices(devices: IDevice[]) {
   })
   return devices
 }
+
+export function safeHostname(name: string, blacklist: string[]) {
+  let index = 1
+  name = name.replace('.local', '')
+  while (blacklist.includes(name.toLowerCase().trim())) {
+    name = name.replace(REGEX_LAST_NUMBER, `-${index}`)
+    index++
+  }
+  return name
+}
+
+export function osName(os?: Ios) {
+  const name = {
+    mac: 'Mac',
+    windows: 'Windows PC',
+    linux: 'Linux system',
+    rpi: 'RaspberryPi',
+  }
+  return os ? name[os] : 'system'
+}
```

Comment:
These would be great candidates for simple unit tests 👍

Reply:
Yea not a bad idea.